history-network.md

```diff
@@ -136,17 +215,41 @@ works for all current transaction types.
 #### Receipts
 
 ```
-selector            = 0x02
-content-key         = Container(chain-id: uint16, block-hash: Bytes32)
-content             = SSZList(ssz-receipt, max-length=MAX_TRANSACTION_COUNT)
-ssz-receipt         = SSZList(encoded-receipt: Byte, max-length=MAX_RECEIPT_LENGTH)
-encoded-receipt     =
+receipt_key = Container(chain_id: uint16, block_hash: Bytes32)
+selector = 0x02
+
+content             = SSZList(ssz_receipt, max_length=MAX_TRANSACTION_COUNT)
+ssz_receipt         = SSZList(encoded_receipt: Byte, max_length=MAX_RECEIPT_LENGTH)
+encoded_receipt     =
   if receipt.is_typed:
     return type_byte + rlp.encode(receipt)
   else:
     return rlp.encode(receipt)
 ```
 
+#### Epoch Accumulator
+
+```
+epoch_accumulator_key = Container(epoch_hash: Bytes32)
+selector = 0x03
+epoch_hash = hash_tree_root(epoch_accumulator)
+
+content = SSZ.serialize(epoch_accumulator)
+```
+
+#### Master Accumulator
+
+```
+master_accumulator_key = Union[None, master_hash: Bytes32]
```

Comment:
Yes, while I'm implementing the part of getting the master accumulator I've also currently not needed the version of getting it from the network with the hash. Not sure if there is a use case for this. (If a node would announce its accumulator hash in ping/pong messages, it could be requested via this hash, but there isn't really a point if you can just ask for latest).

Reply:
So, one interesting, though probably not consequential outcome of removing your union type and replacing it with just `None`, is that we would need to reorder the selectors for the History Network Content Key Union so that the Master Accumulator goes first since its value is just `None`.

Reply:
I went ahead and updated our code to use this approach though we're not handling the case where someone provides a root hash for the master accumulator as there is no use in our current context.  We can update down the line as needed.

Reply:
The only reason I can think up is the case where your local view of the chain is behind someone else's or for some reason you are trying to follow a recent fork that they aren't following.  In that case you'd want to be able to specify the root hash.  I think this is a pretty weak use case.